### PR TITLE
doc: known issues: add known issues for 2.8

### PR DESCRIPTION
**Pull request by theob-pro**
Added the following known issues:
- DRGN-23511
- NCSDK-29354

### CODE REVIEW

**Review by laxiLang — APPROVED** [review]

**Review by alwa-nordic — APPROVED** [review]

**Comment by theob-pro** [comment]
Rebased to fix conflict and pushed a new commit to add NCSDK-29354.

File: doc/nrf/releases_and_maturity/known_issues.rst [hunk 1]
```diff
@@ -273,6 +273,18 @@ DRGN-23518: ACL reassembly may deadlock
...
+
+DRGN-23511: Building of multilink Bluetooth applications fails.
+  This happens when the :kconfig:option:`CONFIG_BT_BUF_RX_COUNT` Kconfig option is not explicitly set.
+
+  **Workaround:** Set the Kconfig value explicitly.
+
+.. rst-class:: v2-8-0
+
+NCSDK-29354: Writing to flash makes Bluetooth hang.
+  This is due to a deadlock between flash erase operation and Bluetooth TX work both done in the system WQ.
```
Comment (peknis) [inline doc/nrf/releases_and_maturity/known_issues.rst:286]:
```suggestion
DRGN-23511: Building of multilink Bluetooth applications fails
  This happens when the :kconfig:option:`CONFIG_BT_BUF_RX_COUNT` Kconfig option is not explicitly set.

  **Workaround:** Set the Kconfig value explicitly.

.. rst-class:: v2-8-0

NCSDK-29354: Writing to flash makes Bluetooth hang
  This is due to a deadlock between flash erase operation and Bluetooth TX work both done in the system WQ.
```
We do not use full stop for the first line. Any workaround to the latter one? And how about the affected platform(s), or are these valid for all?

**Review by peknis — CHANGES_REQUESTED** [review]

File: doc/nrf/releases_and_maturity/known_issues.rst (hunk 1 above)
Comment (wiba-nordic) [inline doc/nrf/releases_and_maturity/known_issues.rst:286]:
```suggestion
  This is due to a deadlock between the flash erase operation and Bluetooth TX work both done in the system workqueue.
```

**Review by wiba-nordic — CHANGES_REQUESTED** [review]

**Review by wiba-nordic — APPROVED** [review]

**Review by peknis — APPROVED** [review]

File: doc/nrf/releases_and_maturity/known_issues.rst (hunk 1 above)
Comment (theob-pro) [reply doc/nrf/releases_and_maturity/known_issues.rst]: Not sure yet, will update accordingly when I get the answer.

File: doc/nrf/releases_and_maturity/known_issues.rst [hunk 2]
```diff
@@ -273,6 +273,18 @@ DRGN-23518: ACL reassembly may deadlock
...
+
+DRGN-23511: Building of multilink Bluetooth applications fails
+  This happens when the :kconfig:option:`CONFIG_BT_BUF_RX_COUNT` Kconfig option is not explicitly set.
+
+  **Workaround:** Set the Kconfig value explicitly.
+
+.. rst-class:: v2-8-0
+
+NCSDK-29354: Bluetooth traffic stalls while performing writing or erasing flash
+  This only occurs when the Bluetooth Controller runs on the same core as the flash operations.
```
Comment (wiba-nordic) [inline doc/nrf/releases_and_maturity/known_issues.rst:286]:
```suggestion
NCSDK-29354: Bluetooth traffic stalls while writing or erasing flash
  This only occurs when the Bluetooth Controller runs on the same core as the flash operations.
```

**Review by wiba-nordic — CHANGES_REQUESTED** [review]

**Review by wiba-nordic — APPROVED** [review]

**Comment by peknis** [comment]
@theob-pro feel free to close this as I just opened a draft PR to update known issues for NCS 2.8.0 (https://github.com/nrfconnect/sdk-nrf/pull/17997)

**Pull request closed without merging**